Makefile

```diff
@@ -79,7 +79,7 @@ bootstrap: tmp build
 integration-test: tmp build
 	cd packages/shopify-cli-extensions; npm link --force
 	./shopify-extensions create testdata/shopifile.integration.yml
-	cd tmp/integration_test; npm link "@shopify/shopify-cli-extensions" && npm install
+	cd tmp/integration_test; npm link "@shopify/shopify-cli-extensions"
```

Comment:
Do you need this change? I think it breaks the tests

Reply:
@vividviolet  this should no longer be needed since we are now running npm install as part of the create command.
I think the tests are failing for different reasons. 
If you delete your `tmp` folder and then run `make integration-test`, you'll see the node_modules are definitely getting created.